mediapipe/tasks/ios/components/processors/sources/MPPClassifierOptions.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Holds settings for any single iOS MediaPipe classification task.
+ */
+NS_SWIFT_NAME(ClassifierOptions)
```

Comment:
We started with having a dedicated `ClassifierOptions` class in Java and Python but ended up flattening the options later in ImageClassifierOptions. Let's do the same for the ObjC API, so instead of having a dedicated `ClassifierOptions` class, we'll inline all these options under ImageClassifierOptions and TextClassifierOptions.
https://github.com/google/mediapipe/blob/c8b8d1fe6b04ef906b7ef3956fdff266c2704228/mediapipe/tasks/java/com/google/mediapipe/tasks/vision/imageclassifier/ImageClassifier.java#L358

@schmidt-sebastian WDYT?

Reply:
That's a good point. We should inline these options, but I think we can commit the code and move it later. At least it will already have been reviewed.